richtextfx-demos/src/main/java/org/fxmisc/richtext/demo/CloneDemo.java

```diff
@@ -0,0 +1,51 @@
+package org.fxmisc.richtext.demo;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import org.fxmisc.richtext.AreaFactory;
+import org.fxmisc.richtext.InlineCssTextArea;
+import org.fxmisc.richtext.StyledTextArea;
+
+public class CloneDemo extends Application {
+
+    public static void main(String[] args) {
+        launch(args);
+    }
+
+    @Override
+    public void start(Stage primaryStage) {
+        String text = "Edit this area and watch the below area update itself.";
+        InlineCssTextArea area = AreaFactory.inlineCssTextArea(text);
+        StyledTextArea<String, String> clone = area.createClone();
+
+        printAreaCarePosition("area", area);
+        printAreaCarePosition("clone", clone);
+
+        area.positionCaret(area.getLength());
+        printAreaCarePosition("area", area);
+        printAreaCarePosition("clone", clone);
+
+        clone.replaceText(0, clone.getLength(), "");
+        printAreaCarePosition("area", area);
+        printAreaCarePosition("clone", clone);
+
+        area.replaceText(0, 0, text);
```

Comment:
This is one bug I ran into: after area sets its text back to `text`, I see only a blank area. The text is there (I only need to navigate to the left in some way or another).

Here's a picture of what that looks like:
![untitled window_009](https://cloud.githubusercontent.com/assets/8413037/11704907/b3126ca2-9ea0-11e5-844f-5f7926fd2019.jpg)

Navigating to the right/down doesn't work (since it's at the end of the area).